File: Modul4/Bahasa_C/PRAK405-2310817210012-Nazmi_Hakim.c
#include <stdio.h>

int main() {
    int n, kelipatan;
    scanf("%d %d", &n, &kelipatan);

    int total = 0;
    for (int i = 1; i <= n; i++) {
        int jbaris = 0;
        for (int j = i; j >= 1; j--) {
            printf("(%d * %d)", j, kelipatan);
            jbaris += j * kelipatan;
            if (j > 1) {
                printf(" + ");
            }
        }
        total += jbaris;
        printf(" = %d\n", jbaris);
    }
    printf("%d\n", total);

    return 0;
}

File: Modul4/Bahasa_Python/PRAK405-2310817210012-Nazmi_Hakim.py
n, kelipatan = map(int, input().split())

total = 0
for i in range(1, n + 1):
    jbaris = 0
    for j in range(i, 0, -1):
        print(f"({j} * {kelipatan})", end='')
        jbaris += j * kelipatan
        if j > 1:
            print(" + ", end='')
    total += jbaris
    print(f" = {jbaris}")

print(total)